snaps/openvino/gpu-support/README.md

```diff
@@ -0,0 +1,59 @@
+# Snap for Testing the OpenVINO Python API
+
+This is a reference snap designed to validate the `openvino-toolkit-2404` content producer snap. It includes CPU, GPU, and NPU support.
+
+## Installation and setup
+
+Build the snap:
+
+```
+snapcraft
+```
+
+Install the snap:
+
+```
+sudo snap install --dangerous ./openvino-python_2024.4.0_amd64.snap
+```
+
+Install content provider snaps to consume from via content interfaces:
+
+```
+sudo snap install --beta intel-npu-driver
+sudo snap install --dangerous ~/openvino-toolkit-2404_2024.4.0_amd64.snap
```

Comment:
i propose to add a comment saying where we can get this snap from

Reply:
The snap is not published yet but I can add a link to the GitHub repo.

Reply:
Done